Tools/setup/macos.sh

```diff
@@ -47,6 +47,8 @@ fi
 
 # Python dependencies
 echo "Installing PX4 Python3 dependencies"
+# We need to have future to install pymavlink later.
+python3 -m pip install future
```

Comment:
Should this add `--user` as well?
```suggestion
python3 -m pip install --user future
```

Reply:
I don't think this is required when you run it with `python3 -m pip` which is the recommended way. (We should change that everywhere...)